assets/settings/default.json

```diff
@@ -148,7 +148,10 @@
     // Whether to show symbols selections in the scrollbar.
     "symbols_selections": true,
     // Whether to show diagnostic indicators in the scrollbar.
-    "diagnostics": true
+    "diagnostics": true,
+    // Scroll sensitivity multiplier. This multiplier is applied 
+    // to both the horizontal and vertical delta values while scrolling.
+    "sensitivity": 1.0
```

Comment:
I don't think `scrollbar` is a good place for this, since you can scroll even with the scrollbar off and

```json
{
"scrollbar": {
  "show": "never",
  "sensitivity": 2.0
}
}
```

seems weird to me. I think a top-level entry would be better. See also:

https://github.com/zed-industries/zed/blob/9efb0e127207f7b5ea0043061f50f1eb7785ccc4/assets/settings/default.json#L170-L172

Reply:
Hi, I have applied your comments and moved this to right next to "vertical_scroll_margin", so it is now at the top level. I also changed the name to "scroll_sensitivity" now that it is no longer underneath the "scrollbar" settings.